sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,6 @@
+select
+  "co"."name" as "country",
+  count("ci"."countryId") as "numCities"
```

Comment:
It'd be clearer to count the `cityId` than the `countryId`, but the output is the same either way.